Task5/css/header/menu/__item.css

```diff
@@ -0,0 +1,36 @@
+.menu__item {
+    display: inline-block;
+
+    margin-right: 30px;
+
+    list-style-type: none;
+}
+
+ul {
```

Comment:
You add global styles for `<ul>` tag here. It should be done (if it was really your intention) in the root css file.